pkg/virt-controller/services/template.go

```diff
@@ -878,32 +878,34 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 	resources.Requests = make(k8sv1.ResourceList)
 	resources.Limits = make(k8sv1.ResourceList)
 
+	// Copy vmi resources requests to a container
```

Comment:
I don't think this is how the `CPUAllocationRatio` is supposed to work. 

I think there are 2 cases:
1. User set a cpu topology. Let's say 1 core.  The expected behavior, in this case, would be to apply the `CPUAllocationRatio`. The resulting VM would not be guaranteed to get the whole cpu time.

2. User wants VM with guaranteed cpu time and therefore sets the `cpu request`. In this case, I don't expect that `CPUAllocationRatio` gets applied.

The problem we need to fix here is to properly distinguish when we are dealing with the 2nd case. User vs limitRange

@vladikr Would you agree?

Reply:
Ok, I understand the scenario you're describing.
First consider the starting point I mentioned above. If the `CPUAllocationRatio` is not designed to be what I said.
Please kindly ask whether it(distinguish User vs limitRange) can be implemented in the new version as soon as possible?